openstates/ak/bills.py

```diff
@@ -71,39 +71,41 @@ def scrape(self, chamber=None, session=None):
             session = self.latest_session()
             self.info('no session specified, using %s', session)
 
-        chambers = [chamber] if chamber is not None else ['upper', 'lower']
-        for chamber in chambers:
-            yield from self.scrape_chamber(chamber, session)
-
-    def scrape_chamber(self, chamber, session):
-        if chamber == 'upper':
-            bill_abbrs = ('SB', 'SR', 'SCR', 'SJR')
-        elif chamber == 'lower':
-            bill_abbrs = ('HB', 'HR', 'HCR', 'HJR')
         bill_types = {'B': 'bill', 'R': 'resolution', 'JR': 'joint resolution',
                       'CR': 'concurrent resolution'}
 
-        for abbr in bill_abbrs:
-            bill_type = bill_types[abbr[1:]]
-            bill_list_url = (
-                'http://www.legis.state.ak.us/basis/range_multi'
-                '.asp?session=%s&bill1=%s1&bill2=%s9999'
-            ) % (session, abbr, abbr)
-            doc = lxml.html.fromstring(self.get(bill_list_url).text)
-            doc.make_links_absolute(bill_list_url)
-            for bill_link in doc.xpath('//table[@align="center"]//tr/td[1]//a'):
-                bill_url = bill_link.get('href')
-                bill_id = bill_link.text.replace(' ', '')
-                yield from self.scrape_bill(chamber, session, bill_id, bill_type,
-                                            bill_url)
+        bill_list_url = (
```

Comment:
for new code, let's favor Python 3 style string formatting

either 'foo {}'.format(session) or f'foo {session}'  :)

Reply:
Fixed.